config/routes.rb

```diff
@@ -1,5 +1,9 @@
 Rails.application.routes.draw do
-  devise_for :users
+
```

Comment:
@romainbazeler please add back devise_for :users to the file

Reply:
model users was not created, that's why the devise route was missing.

Reply:
perfect